nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/test/java/org/apache/nifi/processors/aws/kinesis/stream/pause/TestSwitchableRecordProcessorBlocker.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.processors.aws.kinesis.stream.pause;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.function.Supplier;
+
+public class TestSwitchableRecordProcessorBlocker {
+
+    @Test
+    public void testResumeAfterPause() {
+        final TestSwitchableRecordProcessorBlockerInspector blockerInspector = new TestSwitchableRecordProcessorBlockerInspector();
+
+        final SwitchableRecordProcessorBlocker recordProcessorBlocker = SwitchableRecordProcessorBlocker.create(blockerInspector);
+
+        recordProcessorBlocker.block();
+        final Thread thread = new Thread(createPauseRunnable(recordProcessorBlocker));
+        thread.start();
+
+        blockerInspector.awaitAwaited();
+        Assertions.assertTrue(thread.isAlive());
```

Comment:
The general convention is to use `import static` for assert methods.
```suggestion
        assertTrue(thread.isAlive());
```